doc/user_guide/transformations.rst

```diff
@@ -506,68 +506,52 @@ Interactive
 +++++++++++
 
 To apply a transformation interactively we first parse and analyse the
-code. This allows us to generate a "vanilla" PSy layer. For example ...
-::
+code. This allows us to generate a "vanilla" PSy layer. For example::
 
-    from psyclone.parse.algorithm import parse
-    from psyclone.psyGen import PSyFactory
+    >>> from psyclone.parse.algorithm import parse
+    >>> from psyclone.psyGen import PSyFactory
 
-    # This example uses version 0.1 of the Dynamo API
-    api = "dynamo0.1"
+    # This example uses the LFRIC (dynamo0.3) API
```

Comment:
"Dynamo0.3" for the written name (not the API choice in the code).

Reply:
Done.